index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <hr class="border">
+    <header>
+        <table class="table-1">
+            <tr>
+                <td><img src="https://i.pinimg.com/originals/82/c6/5b/82c65b9bb0a75026fc4c82a438b4cc9b.jpg" alt="h-image"class="image"></td>
+                <td><h1 class="title">Assignment-3</h1></td>
```

Comment:
Add horizontal line above and below the image, as mentioned in the problem statement.